Makefile

```diff
@@ -15,17 +15,18 @@ bats-tests: ## Run Helm chart bats tests.
 # ===========> Control Plane Targets
 
 control-plane-dev: ## Build consul-k8s-control-plane binary.
-	@$(SHELL) $(CURDIR)/control-plane/build-support/scripts/build-local.sh -o $(GOOS) -a $(GOARCH)
+	@$(SHELL) $(CURDIR)/control-plane/build-support/scripts/build-local.sh -o linux -a amd64
```

Comment:
Ignore this file for now. I will merge in Iryna's Makefile/Dockerfile changes when they get merged into main (and clean this up)

Reply:
Sorry! I'll merge this today!